src/workerd/api/tests/compat-flags-test.js

```diff
@@ -0,0 +1,58 @@
+import { ok, strictEqual, throws } from 'node:assert';
+
+const { compatibilityFlags } = globalThis.Cloudflare;
+ok(Object.isSealed(compatibilityFlags));
+ok(Object.isFrozen(compatibilityFlags));
+ok(!Object.isExtensible(compatibilityFlags));
+
+// It shuld be possible to shadow the Cloudflare global
+const Cloudflare = 1;
+strictEqual(Cloudflare, 1);
+
+export const compatFlagsTest = {
+  test() {
+    // The compatibility flags object should be sealed, frozen, and not extensible.
+    throws(() => (compatibilityFlags.no_nodejs_compat_v2 = '...'));
```

Comment:
now that we don't support disable flags, this test throws for a reason that is different than when the test was originally written, and is the same assertion as `not_a_real_compat_flag`. I suggest you either add a comment to say that you are explicitly testing for disable flags which should work as non-existent flags, or remove this line altogether.